index.html

```diff
@@ -49,198 +62,199 @@ <h1>I'm Alier Glad to see you.</h1>
             </li>
           </ul>
         </div>
-      </div>
+      </section>
       <section id="works-section">
-        <div class="card-1 contain">
-          <div class="card-1-container">
-            <div class="image-container"></div>
-
+        <div class="card">
+          <img
+            src="./images/Snapshoot-Portfolio-1.svg"
+            alt="portfolio project"
+          />
+          <div class="content-portfolio">
             <h1>Tonic</h1>
```

Comment:
- [ ] I noticed here the card doesn't have the same width and the same height that exist in the Figma design, Kindly Fix it to be the same according to the requirements.

- [ ] As I can see here the alignment for the elements in the side is not aligned as the Figma and the margin/padding is not as the Figma as well so please fix it.
- [ ]  Also I want to highlight something here I can see that you used here `h1` for Tonic I think it would be better if you used `h2` or `h3` instead of `h1` to follow the semantics of HTML because The H1 tags are one of the most significant SEO elements on the page beside the title tag. and please Fix the Font weight here as well🔥🔥🔥🔥


- your design
![image](https://user-images.githubusercontent.com/58553711/178123008-6df62ae9-fdaf-4a34-a97c-ed79686ef9ab.png)


- Figma design 
![image](https://user-images.githubusercontent.com/58553711/178122919-bebc0cb4-9863-471c-80d6-0a5029f7a77e.png)